templates/.circleci/config.yml

```diff
@@ -132,9 +139,16 @@ jobs:
     steps:
       - attach_workspace:
           at: *workspace
-      - node/install-packages:
-          cache-path: "./node_modules"
-          app-dir: .
+      ## This `npm i --package-lock` step is used so boilerplate can pick up latest version before locking
```

Comment:
the `package-lock.json` changes includes all the dependencies each driver would have 
the different between mysql vs postgres is 200 line differences so it wouldnt really make sense to template